recipes/oras/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "oras" %}
+{% set version = "0.12.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/oras-project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5e19d61683a57b414efd75bd1b0290c941b8faace5fcc9d488f5e4aa674bf03e
+
+build:
+  number: 0
+  script:
+    - mkdir -p $PREFIX/bin                                 # [unix]
+    - make build-mac                                       # [osx and not aarch64]
+    - mv bin/darwin/amd64/oras $PREFIX/bin/oras            # [osx and not aarch64]
+    - make build-linux                                     # [linux and x86_64]
+    - mv bin/linux/amd64/oras $PREFIX/bin/oras             # [linux and x86_64]
```

This looks a like fragile, espeically if tests are skipped. How about:
```suggestion
    - mv bin/linux/amd64/oras $PREFIX/bin/oras             # [linux and x86_64]
    # Make sure a binary has been installed for all unix platforms
    - which oras  # [unix]
```